website/src/pages/docs/server-side-rendering.mdx

```diff
@@ -116,6 +116,36 @@ The `extractor.getScriptTags()` returns a string of multiple `<script>` tags mar
 
 Alternatively the `ChunkExtractor` also has a `getScriptElements()` method that returns an array of React elements.
 
+### Entrypoints
```

Comment:
This is very advanced, can you please put it at the very end of the page? Thanks!

Reply:
Sure, no problem.

Reply:
I moved the section to the end of the page and modified title to give more context.
Thanks for the feedback.